.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to Scooch
+
+## Bugs, Feature Requests
+Submit an [issue](https://github.com/mobify/scooch/issues).
+
+## Pull Requests
+
+1. Make a new branch from `develop`.
+1. Do your work.
+1. Update the README + CHANGELOG.
+1. Open a PR against `develop`.
```

Comment:
Markdown will render them out to correct step numbers, but I'll change it to be more clear.